modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/akula.dm

```diff
@@ -12,7 +12,8 @@
 	)
 	inherent_traits = list(
 		TRAIT_ADVANCEDTOOLUSER,
-		TRAIT_CAN_STRIP
+		TRAIT_CAN_STRIP,
+		TRAIT_CAN_USE_FLIGHT_POTION
```

Comment:
I'm not going to put the suggestion everywhere, but it's good practice to add a comma even after the last element in your list so that the changes the next person has to make are ever so slightly smaller (also helps reduce conflicts but this is a modular file so it doesn't apply as much)

If you could do that in all of the files, that'd be sweet.
```suggestion
		TRAIT_CAN_USE_FLIGHT_POTION,
```